### PR TITLE
Add guide for installation from release tarball

### PR DESCRIPTION
**Pull request by Random-Liu**
Added a instruction to install cri-containerd from release tarball.

Signed-off-by: Lantao Liu <lantaol@google.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ With it, you could run Kubernetes using containerd as the container runtime.
 * It has passed all regular [node e2e tests](https://github.com/kubernetes/community/blob/master/contributors/devel/e2e-node-tests.md).
 ## Installing with Ansible and Kubeadm
 For a multi node cluster installer and bring up steps using ansible and kubeadm refer [here](contrib/ansible/README.md).
+## Installing with Release Tarball and Kubeadm
```
Comment (abhi) [inline README.md]: Custom Installation ? 
For non ansible users , you can pull the cri-containerd and containerd release binaries and deploy kubernetes cluster using kubeadm as described <here>

File: docs/installation.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+## Step 1: Download CRI-Containerd Release Tarball
+Download release tarball for the cri-containerd version you want to install from sources listed [above](#download).
+```console
+$ wget https://storage.googleapis.com/cri-containerd-release/cri-containerd-${VERSION}.tar.gz
+```
+## Step 2: Install CRI-Containerd
+If you are using systemd, just simply unpack the tarball to the root directory:
+```shell
+sudo tar -C / -xzf cri-containerd-${VERSION}.tar.gz
+sudo mkdir -p /etc/cni/net.d
```
Comment (abhi) [inline docs/installation.md]: how about /opt/cni/bin/

File: docs/installation.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
+# Install CRI-Containerd with Release Tarball
+This document provides the steps to install `cri-containerd` and its dependencies with the release tarball, and bring up a Kubernetes cluster using kubeadm.
+
+These steps have been verified on Ubuntu 16.04. For other OS distributions, the steps may differ. Please feel free to file issues or PRs if you encountered any problems on other OS distributions.
```
Comment (abhi) [inline docs/installation.md]: s/encountered/encounter

File: docs/installation.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+If you have other requirements for the binaries, e.g. selinux support, another architecture support etc., you need to build the binaries yourself following [the instructions](../README.md#getting-started-for-developers).
+ 
+### Download
+
+The release tarball could be downloaded from either of the following sources:
+1. Release on github (see [here](https://github.com/kubernetes-incubator/cri-containerd/releases));
+2. Release GCS bucket https://storage.googleapis.com/cri-containerd-release/.
+
+## Step 0: Install Required Libraries
```
Comment (abhi) [inline docs/installation.md]: s/Required/dependent

File: README.md (hunk 1 above)
Comment (Random-Liu) [reply README.md]: Will do.

File: docs/installation.md (hunk 3 above)
Comment (Random-Liu) [reply docs/installation.md]: Will do.

File: docs/installation.md (hunk 4 above)
Comment (Random-Liu) [reply docs/installation.md]: Will do.

File: docs/installation.md (hunk 2 above)
Comment (Random-Liu) [reply docs/installation.md]: ocicni only needs `/etc/cni/net.d` to be there.

File: README.md (hunk 1 above)
Comment (Random-Liu) [reply README.md]: Done.

File: docs/installation.md (hunk 3 above)
Comment (Random-Liu) [reply docs/installation.md]: Done

**Comment by Random-Liu** [comment]
@abhi Addressed comments.

File: docs/installation.md (hunk 2 above)
Comment (abhi) [reply docs/installation.md]: i meant we need to mkdir  /opt/cni/bin/ also :)

Comment (Random-Liu) [reply docs/installation.md]: Hm, really? OK, will add it.

Comment (abhi) [reply docs/installation.md]: @Random-Liu : https://github.com/kubernetes-incubator/cri-containerd/blob/eb2ea51e00fcc37696499718617831109fe737bf/contrib/ansible/tasks/binaries.yaml#L8

Comment (Random-Liu) [reply docs/installation.md]: Done.

**Comment by Random-Liu** [comment]
Apply LGTM based on https://github.com/kubernetes-incubator/cri-containerd/pull/324#pullrequestreview-67476315

**Pull request merged**